usaspending_api/etl/es_award_template.json

```diff
@@ -562,6 +562,30 @@
         "total_covid_outlay": {
           "type": "scaled_float",
           "scaling_factor": 100
+        },
+		"iija_spending_by_defc": {
```

Comment:
Could you fix this indent too?

Reply:
The indentation of the iija fields should match the indentation of the covid fields now.